### PR TITLE
Fixing pathfinding calc when the distance was to small for the bidire…

### LINKED ISSUE #15054: Vehicles in a 3x3 shape with an empty square in the middle - can't navigate
### Issue Summary
If you form vehicles in this shape:

3x3units

And then try to move a unit to the center, it doesn't move.

#### System Information
- **Operating System:** Windows 10
- **.NET / Mono Version:** NET 4.7.1
- **OpenRA Version:** bleed
- **Mod:** RA

#### Additional Information:
- Steps to reproduce
	1. Form 8 vehicles in a 3x3 shape with the middle empty
	2. Move any vehicle to center

### PR DESCRIPTION
**Pull request by teinarss**
Fixes #15054 and #6989

### CODE REVIEW

**Comment by teinarss** [comment]
Updated: fixed it in another way.

File: OpenRA.Mods.Common/Traits/World/PathFinder.cs [hunk 1]
```diff
@@ -70,6 +70,11 @@ public List<CPos> FindUnitPath(CPos source, CPos target, Actor self, Actor ignor
 			if (domainIndex != null && !domainIndex.IsPassable(source, target, li))
 				return EmptyPath;
 
+			var distance = source - target;
+
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/World/PathFinder.cs]: Please remove the empty line here as well.

**Comment by teinarss** [comment]
Updated: removed empty line and using LengthSquared

**Review by chrisforbes — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/73c163b2a267177435122f53f185cb02b293e7c6...e8f3043ef1a90b9308084321b9a452e674fb847a)

**Comment by drogoganor** [comment]
Great work!

**Pull request merged**